core/lib/libmbedtls/src/sub.mk

```diff
@@ -1 +1,16 @@
 srcs-y += tee_lmd_provider.c
+
+# For mbedtls
+cflags-y += -Icore/lib/libmbedtls/src/mbedtls/include
+cflags-y += -D'MBEDTLS_CONFIG_FILE="mbedtls_config.h"'
+cflags-y += -D_FILE_OFFSET_BITS=64 -Wno-old-style-definition -Wno-redundant-decls
```

Comment:
Is `-D_FILE_OFFSET_BITS=64` really needed?

Reply:
Those flags are added in the former work, some of them are unneeded anymore. I will remove the unneeded items.